source/math/FloatMatrix.ooc

```diff
@@ -367,3 +414,7 @@ operator * (left: Float, right: FloatMatrix) -> FloatMatrix {
 	right free(Owner Receiver)
 	result
 }
+
+operator * (left: FloatMatrix, right: Float) -> FloatMatrix {
+	right * left
+}
```

Comment:
No, this should be defined inside the class, like the other operators (preferably next to the other `*` operator implementation):
```ooc
operator * (factor: Float) -> This {
    factor * left
}
```